client/src/api/user/index.ts

```diff
@@ -14,6 +16,20 @@ const getUsers = async (): Promise<Response<UsersResponse>> => {
   return await get(url, failureMessage);
 };
 
+const getPendingContributors = async (): Promise<Response<UsersResponse>> => {
+  const url = buildEndpoint(USER_ENDPOINT, PENDING_CONTRIBUTORS_ENDPOINT);
```

Comment:
Rather than using a constant with a long path, we can use buildEndpoint like this

`const url = buildEndpoint(USER_ENDPOINT, 'contributors', 'pending');`
`const url = buildEndpoint(USER_ENDPOINT, 'staff', 'pending');`